src/devtools/client/debugger/src/components/SecondaryPanes/FrameTimeline.css

```diff
@@ -21,36 +21,45 @@ body.scrubbing {
 .frame-timeline-container {
   border-bottom: 1px solid var(--theme-splitter-color);
   background-color: var(--accordion-header-background);
-  padding: 8px;
+  padding: 12px 10px;
 }
 
 .frame-timeline-bar {
-  background-color: var(--progressbar-background);
-  border: 1px solid var(--theme-splitter-color);
+  background-color: var(--light-color-background-inactive);
```

Comment:
Are sliders supposed to have cursor: pointer?

Reply:
Good call, @jkup. Pointer on hover, and Grabbing on mousedown.